setup.py

```diff
@@ -4,17 +4,26 @@
 
 from setuptools import setup, find_packages
 
+
+def get_requirements(env):
+    with open('requirements-{}.txt'.format(env)) as f:
+        return [x.strip() for x in f.read().split('\n') if not x.startswith('#')]
+
+
 setup(
     name='shipitapi',
     version='1.0.0',
     description='Lib package to facilitate access to Ship IT API operations',
     url='https://github.com/MihaiTabara/shipitapi',
     author='Mihai Tabara',
     author_email='mtabara@mozilla.com',
+    install_requires=get_requirements('prod'),
+    extras_require={
```

Comment:
You can remove this. No need for it since any developer can also run `tox` locally and get the same environment of testing as is on Travis CI.